src/twisted/conch/ssh/keys.py

```diff
@@ -298,7 +373,7 @@ def _fromString_PRIVATE_OPENSSH(cls, data, passphrase):
             * a passphrase is not provided for an encrypted key
         """
         lines = data.strip().split(b'\n')
-        kind = lines[0][11:14]
+        kind = lines[0][11:14].replace(b' ', b'')
```

Comment:
This is required for EC keys,  `-----BEGIN EC PRIVATE KEY-----[11:14]` gives `EC `. It wasn't required before because `RSA` and `DSA` are of length 3